bin/ncp/NETWORKING/namecheapDNS.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Namecheap DNS updater client installation on Raspbian 
+#
+# Copyleft 2020 by ndunks and Huizerd
+# GPL licensed (see end of file) * Use at your own risk!
+#
+# Based on:
+# - https://gist.github.com/ndunks/c756030c0757b667c9a478c97ca5a9b7
+# - https://www.namecheap.com/support/knowledgebase/article.aspx/29/11/how-do-i-use-a-browser-to-dynamically-update-the-hosts-ip
+#
+# Further steps to be taken:
+# - Buying a Namecheap domain
+# - https://www.namecheap.com/support/knowledgebase/article.aspx/595/11/how-do-i-enable-dynamic-dns-for-a-domain/
+# - https://www.namecheap.com/support/knowledgebase/article.aspx/43/11/how-do-i-set-up-a-host-for-dynamic-dns
+
+
+install()
+{
+  apt-get update
+  apt-get install --no-install-recommends -y dnsutils
+}
+
+configure() 
+{
+  local updateurl=https://dynamicdns.park-your-domain.com/update
+  local url="${updateurl}?host=${HOST}&domain=${DOMAIN}&password=${PASSWORD}"
+
+  [[ $ACTIVE != "yes" ]] && { 
+    rm -f /etc/cron.d/namecheapDNS
+    service cron restart
```

Comment:
Is it possible to use `systemctl restart crond` here?

Reply:
Possibly? I don't know for sure, I just copied the majority of this file from https://github.com/nextcloud/nextcloudpi/blob/master/bin/ncp/NETWORKING/freeDNS.sh.

Reply:
`crond` gave an error, should be `cron` I think